pkg/controller/jobs/generic/config_test.go

```
@@ -0,0 +1,145 @@
+		t.Run(tt.name, func(t *testing.T) {
+			err := cm.LoadConfigurations(tt.configs)
+			if (err != nil) != tt.wantErr {
+				t.Errorf("ConfigManager.LoadConfigurations() error = %v, wantErr %v", err, tt.wantErr)
+			}
+		})
+	}
+}
+
+func TestConfigManager_GetAdapter(t *testing.T) {
+	cm := NewConfigManager()
+	
```

golangci-lint (2.2.2)

File is not properly formatted

(gci)